src/threat_model/threat_model.md

```diff
@@ -133,6 +156,8 @@ This attacker uses the existing Listener endpoint, created by the service, to co
 This attacker uses the existing Listener endpoint, created by the service, to communicate with the
 client. It can also create a spoofed endpoint, mimicking the service's one.
 
+For protection of the clients' assets, please check the Parsec Clients Threat Models.
```

Comment:
😆 check the what?

Reply:
Well we can put the links once we have them :D